portal/mock-server/src/router.ts

```diff
@@ -29,6 +29,14 @@ router.get('/iterations', async (req: Request, res: Response) => {
   }, 1500);
 });
 
+router.get('/message_sizes', async (req: Request, res: Response) => {
```

Comment:
why the message size came from different API and not a part of `all-experiments.json`?

Reply:
This API is related to the homepage, when we get the predefined list.
Nothing related to the all experiments.